+++
title = "Bacterial genome-wide association study to identify genetic variants linked to *in vitro* phenotypes"
date = 2018-09-13T00:00:00  # Schedule page publish date.
draft = false

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
#time_start = 2030-06-01T13:00:00
#time_end = 2030-06-01T15:00:00

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Katie Saund"]

# Abstract and optional shortened version.
abstract = "*C. difficile* infections can lead to drastically different outcomes: from a short infection with a full recovery to severe disease with recurrent infections or death. While undoubtedly much of the variation in disease outcomes are due to patient factors, our aim is to identify the bacterial genetic determinants that contribute to variable patient outcomes. At this point, the challenge of predicting patient outcome from the WGS of the infection strain is too complex, however we believe we can make meaningful progress towards this goal because of the promising results from our bacterial genome-wide association study. We have a cohort of over 100 clinical *C. difficile isolates* which have been thoroughly phenotyped in a battery of in vitro assays. These phenotypes are likely directly relevant to disease and transmission, such as toxin production and germination efficiency. Using a combination of the genetic variation encoded in the whole sequences of these isolates & the in vitro phenotypes, we performed multiple different association tests to identify the genetic loci correlated with the phenotypes. The results from our study appear robust as we have recapitulated known genotype-phenotype relationships, such as *tcdA* mutations associating with toxin activity and *spo0A* associating with spore viability. Critically, we have identified many novel loci associated with these clinically relevant phenotypes. We intend to validate some of these new associations via genetic manipulations in *C. difficile* laboratory strains."
abstract_short = ""

# Name of event and optional event URL.
event = "2nd Annual Meeting of the Systems Biology and Antibacterial Resistance Program"
event_url = ""

# Location of event.
location = "La Jolla, CA"

# Is this a selected talk? (true/false)
selected = true

# Projects (optional).
#   Associate this talk with one or more of your projects.
#   Simply enter your project's filename without extension.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
#   Otherwise, set `projects = []`.
projects = []

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = []

# Links (optional).
url_pdf = ""
url_slides = "" #"slides/example-slides"
url_video = ""
url_code = ""

# Does the content use math formatting?
math = false

# Does the content use source code highlighting?
highlight = false

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = ""
caption = ""

+++

{{% alert note %}}
Click on the **Slides** button above to view the built-in slides feature.
{{% /alert %}}

Slides can be added in a few ways:

- **Create** slides using Academic's *Slides* feature and link using `url_slides` parameter in the front matter of the talk file
- **Upload** an existing slide deck to `static/` and link using `url_slides` parameter in the front matter of the talk file
- **Embed** your slides (e.g. Google Slides) or presentation video on this page using [shortcodes](https://sourcethemes.com/academic/docs/writing-markdown-latex/).

Further talk details can easily be added to this page using *Markdown* and $\rm \LaTeX$ math code.
